verilog/tools/lint/README.md

```diff
@@ -213,6 +215,15 @@ auto-generated.
 
 The name of the rule to waive is at the end of each diagnostic message in `[]`.
 
+These waiver rules can be automatically generated with the AutoFix mode `generate-waiver` to stdout,
+or to a waiver file if a path is passed to `--autofix_output_file`,
+e.g. 
+`verible-verilog-lint file.sv --autofix=generate-waiver --autofix_output_file violations.waiver`
+Generated waiving rules will follow this format:
```

Comment:
The command here is a bit long and it wraps. Let's format this similarly to [Rule configuration](https://github.com/chipsalliance/verible/blob/aec56ff69188b71ce2985bac765dc33dc0a09536/verilog/tools/lint/README.md#rule-configuration) section:


~~~
These waiver rules can be automatically generated with the AutoFix mode `generate-waiver` to stdout,
or to a waiver file if a path is passed to `--autofix_output_file`.

```bash
verible-verilog-lint --autofix=generate-waiver --autofix_output_file=violations.waiver ...
```

Generated waiving rules will follow this format:
~~~